Revit_Core_Engine/Query/GetGroupTypeId.cs

```diff
@@ -36,15 +36,15 @@ public static partial class Query
         [Description("Gets the BuiltInParameterGroup of a Revit parameter definition.")]
         [Input("def", "A Revit parameter definition to get BuiltInParameterGroup for.")]
         [Output("parameterGroup", "BuiltInParameterGroup of a Revit parameter definition.")]
-        public static BuiltInParameterGroup GroupTypeId(this Definition def)
+        public static BuiltInParameterGroup GetGroupTypeId(this Definition def)
         {
             return def.ParameterGroup;
         }
 #else
         [Description("Gets the GroupTypeId of a Revit parameter definition.")]
         [Input("def", "A Revit parameter definition to get GroupTypeId for.")]
         [Output("groupTypeId", "GroupTypeId of a Revit parameter definition.")]
-        public static ForgeTypeId GroupTypeId(this Definition def)
+        public static ForgeTypeId GetGroupTypeId(this Definition def)
```

Comment:
I think this method could be removed altogether, so that you can simply call `Definition.GetGroupTypeId()` everywhere, with usings being `BH.Revit.Engine.Core` for Revit 2024 and lower and `Autodesk.Revit.DB` for 2025 and above